pkgs/development/libraries/dbus/default.nix

```diff
@@ -50,7 +60,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" "lib" "doc" "man" ];
 
-  nativeBuildInputs = [
+  nativeBuildInputs = lib.optionals stdenv.hostPlatform.isStatic
```

Comment:
Given that you changed the patching to am files, these will need to be unconditional.

Reply:
Fixed.